scripts/catch/1.9.6/.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: generic
+
+sudo: false
+
+matrix:
+  include:
+    - os: osx
+      compiler: clang
+      osx_image: xcode8.2
+    - os: linux
+      compiler: clang
+      addons:
+        apt:
+          sources: [ 'ubuntu-toolchain-r-test' ]
+          packages: [ 'libstdc++-5-dev' ]
+
```

Comment:
Woops, grabbed this from tippecanoe accidentally. Good catch! Updated in https://github.com/mapbox/mason/pull/458/commits/def305dee4ac30bc037a6631e999b773baefa6f1